src/components/camera/Client.ts

```diff
@@ -0,0 +1,79 @@
+import { type Camera, MimeType, Properties } from './Camera'
+import { CameraServiceClient } from '../../gen/component/camera/v1/camera_pb_service.esm'
+import type Client from '../../Client'
+import type { HttpBody } from '../../gen/google/api/httpbody_pb'
+import pb from '../../gen/component/camera/v1/camera_pb.esm'
+import { promisify } from '../../utils'
+
+export class CameraClient implements Camera {
+  private client: Client | undefined
+  private readonly name: string
+
+  constructor (client: Client, name: string) {
+    this.client = client
+    this.name = name
+  }
+
+  private get cameraService () {
```

Comment:
Im curious, what is the use case for this method?

Reply:
related to https://github.com/viamrobotics/viam-typescript-sdk/pull/6#discussion_r1044666570 - this method encapsulates creating a `CameraServiceClient` from an existing client connection.

Reply:
I will add a utility for this logic since each component wrapper will need it - assuming we are okay sticking with this pattern

Reply:
I think it makes sense to either add it to utility or to have a component interface that has a generic version of this. Then we can include any shared component functions in the component interface and have new components get it for free. Something like this https://github.com/viamrobotics/viam-python-sdk/blob/main/src/viam/components/component_base.py